docker/release

```diff
@@ -9,9 +9,9 @@ ENV PYTHONUNBUFFERED=1
 RUN apt update && apt install -y build-essential
 
 # db2 requirement
-RUN apt install -y libxml2 libmagic1 || true
+RUN apt install -y libxml2 libmagic1 libkrb5-dev || true
 
-RUN python -m pip install --prefer-binary --no-cache-dir --upgrade pip==22.1.2 && \
+RUN python -m pip install --prefer-binary --no-cache-dir --upgrade pip==23.1.2 && \
```

Comment:
Should we always use the latest pip or the >= 23 version?

Reply:
I installed latest, then checked what version it was and fixed in docker image because it was already fixed.
It seems to me it is safer to have the same version I tested the image but we can also change to latest version